sos/report/plugins/rear.py

```diff
@@ -20,12 +20,14 @@ class Rear(Plugin, RedHatPlugin):
     def setup(self):
         # don't collect recovery ISOs or tar archives
```

Comment:
Good idea to forbid whole output dir (as far as I read man pages and `OUTPUT_URL=file://`). Is the "recovery ISOs or tar archives" still valid description of content of the dir?

Reply:
So the iso will pretty much always be in the output directory, but the backup tar file is rarely stored locally since it could fill the filesystem up. So I figured just excluding everything in that directory just in case.